core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/AkkaRunner.scala

```diff
@@ -201,6 +203,11 @@ object AkkaRunner extends Runner[Deployment] {
         .addVolume(secretVolume)
         .addVolume(Runner.DownwardApiVolume)
 
+    val podSpecMoreVolumes = configSecretVolumes.foldLeft[Pod.Spec](podSpec) {
```

Comment:
Can we think of a more descriptive name here? Just more volumes do not tell me much when reading this.

Reply:
Just FYI This is a nice place for `|>` but yeah to add an implicit etc for `Any`.. Scala 2.13 has it it seems https://www.scala-lang.org/api/current/scala/util/ChainingOps.html

Reply:
Having to come up with a name for the thing just because you want to chain, is a bit irritating

Reply:
done